TripPlanner - Project/TripPlannerFrontend/src/components/Buttons/ButtonPrimaryLink.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+
+const ButtonLink = styled.button`
+  padding: 10px 18px 10px 18px;
+  width: 105px;
+  height: 34px;
+  background-color: transparent;
+  border: 0;
+  border-radius: 6px;
+  color: #F53314;
+  cursor: pointer;
+  left: calc(50% - 105px/2);
+  top: calc(50% - 34px/2 + 54px);
+  font-family: Work Sans, sans-serif;
```

Comment:
You can extract this style to global one, to not repeat yourself (keeping code DRY). 

See `src/GlobalStyle.ts`